trace.c

```diff
@@ -61,7 +65,7 @@ long int RETRACE_IMPLEMENTATION(ptrace)(enum __ptrace_request request, ...)
 		case PT_READ_D:
                         request_str = "PT_READ_D";
                         break;
-		case PT_READ_U:
+		case 3: /* PT_READ_U: */
```

Comment:
PT_READ_U was removed from FreeBSD.

Historically PT_READ_U has been 3 since ptrace() was first written.

The fact that PT_READ_U is no longer defined in FreeBSD doesn't mean that there aren't binaries with this compiled in. 

I think hardcoding the 3 here is perfectly acceptable instead of creating a #ifdef mess